tsconfig.base.json

```diff
@@ -4,7 +4,7 @@
     "moduleResolution": "nodenext",
     "moduleDetection": "force",
     "target": "es6",
-    "lib": ["es2020"],
+    "lib": ["es2021"],
```

Comment:
needed to bump this to use `.replaceAll`.  Seemed to work locally; we'll see if it passes on our legacy VSCode test

Reply:
Looks like it's fine even on legacy VSCode